Packages/com.unity.multiplayer.samples.coop/package.json

```diff
@@ -9,7 +9,7 @@
   "dependencies": {
     "com.unity.learn.iet-framework": "1.2.1",
     "com.unity.multiplayer.tools": "1.0.0",
-    "com.unity.netcode.gameobjects": "1.0.2",
+    "com.unity.netcode.gameobjects": "1.1.0",
```

Comment:
Adding note here to not merge until we have manifest and package.json updates done. (done once NGO 1.1.0 is out)